docs/plugins/virtual_environment_docs/2-deploying-your-plugin.md

```diff
@@ -0,0 +1,109 @@
+# Deploying your plugin  
+
+This guide explains some of the techniques you can use to deploy your plugin. 
+
+## This guide covers:   
+* [Overview of PyPI and Anaconda](#overview-of-pypi-and-anaconda)
+* [Building your package](#building-your-package)
+* [Deploying plugins to PyPI](#deploying-plugins-to-pypi)
+    - [Manually via twine](#manually-via-twine)
+    - [Automatically via GitHub actions](#automatically-via-github-actions)
+* [Deploying plugins to Anaconda](#deploying-to-anaconda)
+
+
+## Overview of PyPI and Anaconda
+PyPI and Anaconda are two options for how you distribute your package and allow your users to more easily find and install it.  Try to deploy to both! But for now, try to at least use PyPI. You can always also provide your users with manual installation instructions (e.g. if you want them to use `conda` or have specific dependencies).
+
+### Building your package 
+**sdist** means source distribution. An **sdist** includes all of the files that are required to *build* your package. An **sdist** may require specific additional software (e.g. compilers) to actually build.
```

Comment:
```suggestion
`sdist` means source distribution. An `sdist` includes all of the files that are required to *build* your package. An `sdist` may require specific additional software (e.g. compilers) to actually build.
```

I think we should swap all bolded references to `sdist` and `wheel` with backticks.